### ISSUE TITLE
Hub 01 intercom teleporting

### ISSUE THREAD

**Issue by Alterforce** [CleverRaven/Cataclysm-DDA#34832]
# Describe the bug

 the intercom NPC for hub 01 teleports outside of the room behind the intercom terrain.  This happens upon completing the mission "Steal a dead man's mind" and then moving to another tile on the map.

# Steps To Reproduce

1. Accept the mission "Steal a dead man's mind".
2. Complete the mission "Steal a dead man's mind".
3. Move to another tile on the map
4. The NPC teleports to a different area.

# Expected behavior

The NPC stays in the same area.

# Screenshots

**If applicable**, add screenshots to help explain your problem.

![intercom tport 1](https://user-images.githubusercontent.com/43394039/67085034-af6a9480-f1d0-11e9-9fd5-b90c583cb07d.png)
"Additional context" --> Current mission.

![intercom tport 2](https://user-images.githubusercontent.com/43394039/67085095-cc06cc80-f1d0-11e9-99d7-336c23fa149a.png)
"Additional context" --> Position of NPC before completing mission.

![intercom tport 3](https://user-images.githubusercontent.com/43394039/67085145-e04ac980-f1d0-11e9-9412-fde4c4b8f241.png)
"Additional context" -->  Mission completed.

![intercom tport 4](https://user-images.githubusercontent.com/43394039/67085636-be057b80-f1d1-11e9-83fe-f8bdcf214f6d.png)
"Additional context" --> Position of NPC after completing mission.

# Versions and configuration

 - OS: Windows 10
 - Game Version: 0.D-8616-gc7fc9cb (tiles)
 - Graphics version: Terminal
 - Mods loaded: 
Dark Days Ahead
Disable NPC Needs
Aftershock
CRIT Expansion Mod
Magiclysm
DeadLeaves' Fictional Guns
Garden Pots
Icecoon's Arsenal
Medieval and Historic Classes
Mythological Replicas
DinoMod
Modular Turrets
Salvaged Robots
Hydroponics
Mutant NPCs
No Rail Stations
Folding Parts Pack
Vehicle Additions Pack
Tanks and Other Vehicles
Safe Autodoc
Manual Bionic Installation
Stats Through Skills

# Additional context
As the NPC is hostile, he will attempt to attack the player.

**Comment by zeggz** [comment]
Can confirm this bug. In my game the intercom teleported behinf the hub bulding in the wood. But it happen as soon as the free merchants caravan spawned  (after the second mission i think, can't remember exactly, it's been a while). Just after the completition of the quest there have been a huge noise (deafening my character) and the intercom npc was moved to the woods (and hostile). I have tryed to reload the game and the same chain of events happenened. Maybe it's related to this noise making the npc investigating the noise and consequently creating this bug, i don't know.

I have observed a similar issue with npcs in the refugee centers, when half of them teleported outisde the bulding. It was after one of the zombie was misplaced outside the bulding during the quest to clear the room with zombies. 

Hope these informations help someone figuring out what's the cause of theses really annoying bugs.
